_extras/guide.md

```diff
@@ -1,6 +1,78 @@
 ---
 title: "Instructor Notes"
 ---
-FIXME
+
+This page includes some tips, reminders and advice for giving this tutorial.
+## Tutorial Content
+
+Well ahead of the planned tutorial date, please go over the available 
+tutorial material 
+in your own time at least once to note specifics such as areas where 
+your cohort may need extra time, any changes you want to make or 
+additional examples you would like to include. 
+
+If you want to add to or adapt the material, you can create your own copy to 
+modify by forking the repository. See here for a guide  on [how to fork a repository](https://docs.github.com/en/enterprise/2.13/user/articles/fork-a-repo).
+
+Please use our guide for [Contributing](CONTRIBUTING.md) when creating your 
+own material just in case you would like to add to the current 
+repository in the future.
+
+Once you are satisified with the material you would like to use and have 
+an estimate of the time required, please proceed with meeting logistics.
+ 
+## Meeting Logistics
+
+Decide if the tutorial will be held on site or online. If on site, make 
+sure of venue availability before finalizing the date. If online, set up connection details.
+
+Once you have agreed to a date, and attendees have signed up, send them an email
+with the following information: 
+ - a link to the [pre-tutorial instructions page](setup.md)
+ - a contact email address so they can post any questions that arise during setup
+ - Venue details (if on site) or meeting connection details (if online).   
+
+Also allow for sufficient time to get access to any 
+accounts etc. which could take more than a few days in some cases (a good 
+thumb rule is to have the time to send a reminder email a few days after
+the first email, 
+so that participants will still have time after the reminder to complete 
+setup). 
+
+Plan for the tutorial to last at least two hours (this could vary based on
```

Comment:
We know that our tutorial takes two hours, but does this tutorial take two hours as well?